### PR TITLE
Addition of URLS

### PR DESCRIPTION
**Pull request by BenjaminBrashaw**

### CODE REVIEW

File: .pa11yci [hunk 1]
```diff
@@ -1,6 +1,12 @@
 {
+    "defaults": { 
+        "chromeLaunchConfig":
+         { "args": ["--no-sandbox"] }
+    },
+    
     "urls": [
-        
+        "https://jptw3amsi2.eu-west-1.awsapprunner.com/homepage",
+        "https://jptw3amsi2.eu-west-1.awsapprunner.com/jobRoles"
```
Comment (boydstepkainos) [inline .pa11yci:9]: are the casing of both urls definitely correct?

File: .github/workflows/pa11y.yml [hunk 2]
```diff
@@ -1,9 +1,8 @@
 name: pa11y
 
 on:
-  pull_request:
-    branches:
-      - main
+  schedule:
+    - cron: '5 4 * * *'
```
Comment (boydstepkainos) [inline .github/workflows/pa11y.yml:8]: The requirement is "integrate pa11y with CI pipeline to run on each UI pull request", which this isn't

Comment (boydstepkainos) [reply .github/workflows/pa11y.yml:8]: Can you request a review from Jason, please :)

Comment (BenjaminBrashaw) [reply .github/workflows/pa11y.yml:8]: made the edits for this

Comment (JasonWKainos) [reply .github/workflows/pa11y.yml:8]: Sit down and speak with Kaja, they have their pa11y system working with CI pipeline. Make sure you understand how it is implemented

**Review by ReannaMcE — APPROVED** [review]

**Review by shaunganley — APPROVED** [review]

**Pull request merged**